### PR TITLE
Beat board

### LINKED ISSUE #944: Beat Board
# Aim

- Allows users to create music through a sound board

# Details

- GUI showing a grid of the different sounds that can be produced 
- Users can either click the pads or press keys on their keyboard to produce the sound

### PR DESCRIPTION
**Pull request by albertzeap**
# Description

Created a beat board which allows the user to activate specific sounds based on the
button or key pressed.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

## Project Metadata
`If there is no-file/nothing to fill the below fields with, then type none:`
`Example:` If no requirements.txt needed/present then type in `Requirements` none => `Requirements: none`

Category:
- [ ] Calculators
- [ ] AI/ML
- [ ] Scrappers
- [ ] Social Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing 
- [ ] Games 
- [ ] Networking
- [ ] OS_Utilities 
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [x] Fun
- [ ] Others

Title: Beat_Board

Folder: Beat-Board

Requirements: requirements.txt

Script: beatBoard.py

Arguments: none

Contributor: albertzeap

Description: A small GUI program that allows the user to create custom sounds through mouse clicks or key presses.

### CODE REVIEW

File: Beat-Board/README [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Authors: Albert Paez | albertzeap@gmail.com
```
Comment (kaustubhgupta) [inline Beat-Board/README:1]: This readme is not in accordance with the repository. Have a look at this template:
https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md

File: Beat-Board/beatBoard.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+# All the locations of the sounds
+kickLocation  = 'Sounds/Kick.wav'
+hiHatLocation = 'Sounds/hiHat.wav'
+snareLocation = 'Sounds/snare.wav'
+pad1Location  = 'Sounds/Pad1.wav'
+pad2Location  = 'Sounds/Pad2.wav'
+pad3Location  = 'Sounds/Pad3.wav'
+pad4Location  = 'Sounds/Pad4.wav'
+pad5Location  = 'Sounds/Pad5.wav'
+pad6Location  = 'Sounds/Pad6.wav'
```
Comment (kaustubhgupta) [inline Beat-Board/beatBoard.py:27]: fix the paths for these. One example: 

`./Beat-Board/Sounds/Kick.wav`

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Beat-Board/README (hunk 1 above)
Comment (albertzeap) [reply Beat-Board/README]: I created a new readme that follows the template

File: Beat-Board/beatBoard.py (hunk 2 above)
Comment (albertzeap) [reply Beat-Board/beatBoard.py]: I have also updated the paths as well

**Review by kaustubhgupta — APPROVED** [review]
So cool project! I loved playing with the beats!

**Review by santushtisharma10 — APPROVED** [review]
@albertzeap 💯

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
@albertzeap Add the requirements.txt file and update the PR template with project metadata. Refer to this PR for implementation: https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/1002

**Comment by albertzeap** [comment]
> @albertzeap Add the requirements.txt file and update the PR template with project metadata. Refer to this PR for implementation: #1002

I edited the PR to match the template of the implementation. I was wondering what needed to be in the requirements.txt file? Would it just be the playsound module that I used?

**Comment by kaustubhgupta** [comment]
> > @albertzeap Add the requirements.txt file and update the PR template with project metadata. Refer to this PR for implementation: #1002
> 
> I edited the PR to match the template of the implementation. I was wondering what needed to be in the requirements.txt file? Would it just be the playsound module that I used?

Yes, we need the requirements file for master script

**Comment by albertzeap** [comment]
> > > @albertzeap Add the requirements.txt file and update the PR template with project metadata. Refer to this PR for implementation: #1002
> > 
> > 
> > I edited the PR to match the template of the implementation. I was wondering what needed to be in the requirements.txt file? Would it just be the playsound module that I used?
> 
> Yes, we need the requirements file for master script

Alright I just created the requirements.txt

File: Beat-Board/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+The following modules are required to run beatBoard.py
```
Comment (kaustubhgupta) [inline Beat-Board/requirements.txt:1]: wrong way of defining requirements.txt file. See this example: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**